io/src/main/scala/fs2/io/ToInputStream.scala

```diff
@@ -0,0 +1,259 @@
+package fs2.io
+
+
+import java.io.{IOException, InputStream}
+
+
+import fs2._
+import Chunk.Bytes
+import util.{Async, Attempt}
+import async.mutable
+import Async.Change
+
+import scala.concurrent.SyncVar
+
+
+/**
+  * Conversion of Stream[F,Byte] to java.io.InputStream
+  */
+object ToInputStream {
```

Comment:
Yeah, I think it would be good to have all the java input/output stream stuff in the same file -- I think that's more consistent with the style of the code base as-is.